nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -6,86 +6,174 @@ When creating a Dynamic Group, one must select a Content Type to which it is ass
 
 Once created the Content Type for a Dynamic Group may not be modified as this relationship is tightly-coupled to the available filtering parameters. All other fields may be updated at any time.
 
-## Creating Dynamic Groups
+## Introduction
+
+### Creating Dynamic Groups
 
 Dynamic Groups can be created through the UI under _Organization > Dynamic Groups_ and clicking the "Add" button, or through the REST API.
 
 Each Dynamic Group must have a human-readable **Name** string, e.g. `devices-site-ams01` and a **Slug**, which should be a simple database-friendly string. By default, the slug will be automatically generated from the name, however you may customize it if you like. You must select a **Content Type** for the group that determines the filtering parameters available include objects as member into the group. Finally, you may also assign a an optional human-friendly **Description** (e.g. "Devices in site AMS01").
 
-Once a new Dynamic Group is created, the **Filter Fields** or **Child Groups** may be specified. This must be done after the group has been created by clicking the "Edit" button.
+Once a new Dynamic Group is created, the group can be configured by clicking the "Edit" button to specify **Filter Fields** or **Child Groups** to use to narrow down the group's member objects. More on this below.
 
 !!! warning
     The content type of a Dynamic Group cannot be modified once created, so take care in selecting this initially. This is intended to prevent the possibility of inconsistent data and enforces the importance of thinking about the data model when defining a new Dynamic Group.
 
 ### Working with Dynamic Groups
 
-Dynamic Groups can be accessed from the primary Dynamic Groups landing page in the web interface under the _Organization > Dynamic Groups_ menu. From there you may view the list of available groups, search or filter the list, view or edit an individual group, or bulk delete groups. Additionally if a group's filter has matching members, the number of members may be clicked to take you to a filtered list view of those objects.
+Once created and configured, Dynamic Groups can be accessed from the primary Dynamic Groups landing page in the web interface under the _Organization > Dynamic Groups_ menu. From there you may view the list of available groups, search or filter the list, view or edit an individual group, or bulk delete groups. Additionally if a group's filter has matching members, the number of members may be clicked to take you to a filtered list view of those objects.
```

Comment:
Let's change "filtered list view" to the "list of members for that dynamic group". 

See: https://github.com/nautobot/nautobot/pull/2200/commits/d5c124fa7ff52d00151e3203b1d5aaba3638669d